modyn/tests/selector/internal/grpc/test_selector_grpc_servicer.py

```diff
@@ -4,21 +4,16 @@
 from typing import Iterable
 from unittest.mock import MagicMock, patch
 
-from modyn.selector.internal.grpc.generated.selector_pb2 import (  # noqa: E402, E501, E611
+from modyn.selector.internal.grpc.generated.selector_pb2 import (  # noqa: E402, E501, E611; StrategyConfig,
```

Comment:
I think adding this probably did not happen on purpose

Reply:
done.